android/engine/src/main/java/org/smartregister/fhircore/engine/auth/AccountAuthenticator.kt

```diff
@@ -183,6 +186,16 @@ constructor(
     }
   }
 
+  fun getPractitionerDetailsFromAssets(): org.hl7.fhir.r4.model.Bundle {
+    val jsonPayload =
+      context.assets.open("sample_practitionar_payload.json").bufferedReader().use { it.readText() }
```

Comment:
Please add the literal string to the constants

Reply:
Done